fastseq/optimizer/fairseq/generate.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, args, task, message_queue):
 
         self.args = args
         self.message_queue = message_queue
-        self.has_target = True
+        self.has_target = False
```

Comment:
In line 128, there is another has_target by default set to be True, i.e., "self.has_target = True". Do we need to change that one as well?

Reply:
Good catch. That one is not used by other places. Line-140-247 creates an internal variable. Let me remove the one in line 128.